test/api/v3/integration/groups/POST-groups_groupId_join.test.js

```diff
@@ -16,7 +16,39 @@ describe('POST /group/:groupId/join', () => {
     });
   });
 
-  context('Accepting invitation to a private guild', () => {
+  context('Joining a public guild', () => {
+    let user, joiningUser, publicGuild;
+
+    beforeEach(async () => {
+      let {group, groupLeader} = await createAndPopulateGroup({
+        groupDetails: {
+          name: 'Test Guild',
+          type: 'guild',
+          privacy: 'public',
+        },
+      });
+
+      publicGuild = group;
+      user = groupLeader;
+      joiningUser = await generateUser();
```

Comment:
The test bellow checks that the user can still join even if he wasn't invited. I'm not sure if I should add a test that you can still join a public guild if you were invited, thoughts?